.github/ISSUE_TEMPLATE/rdsm-ruby-client-issue-template.md

```diff
@@ -0,0 +1,49 @@
+---
+name: RDSM Ruby Client Issue Template
+about: Template for new Issues for the RD Station Ruby Client Project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+# Prerequisites
+
+Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**
+
+- [ ] I am running the latest version
+- [ ] I checked the documentation and found no answer
+- [ ] I checked to make sure that this issue has not already been filed
+- [ ] I'm reporting the issue to the correct repository (for multi-repository projects)
+
+# Expected Behavior
+
+Please describe the behavior you are expecting
+
+# Current Behavior
+
+What is the current behavior?
+
+# Failure Information (for bugs)
+
+Please help provide information about the failure if this is a bug. If it is not a bug, please remove the rest of this template.
```

Comment:
I think we can use either `Please help to provide...` or `Please provide...`

Reply:
Done ;)